app/views/layouts/_header.html.erb

```diff
@@ -49,7 +49,7 @@
               <span class="nav-link-title">Help</span>
             </a>
             <div class="dropdown-menu">
-              <a class="dropdown-item" href="https://appmaintainers.com">
+              <a class="dropdown-item" href="https://appmaintainers.com" target="_blank">
```

Comment:
Megtaláltad a jó megoldást! :blush:   
Annyit szeretnék még kérni, hogy van egy másik html attribútum is, amit érdemes odaadni azoknak a linkeknek, amik új ablakot nyitnak meg. Ez a következő:
```
rel="noreferrer"
```

See for details: https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable  
> Essentially, adding rel="noopener noreferrer" to links protects your site's users against having the site you've linked to potentially hijacking the browser (via rogue JS).